README.rst

```diff
@@ -416,16 +416,17 @@ Want to intercept standard `logging` messages toward your `Loguru` sinks?
 ::
 
     class InterceptHandler(logging.Handler):
-        def emit(self, record):
+        def emit(self, record: logging.LogRecord) -> None:
             # Get corresponding Loguru level if it exists.
+            level: str | int
```

Comment:
The typehint can actually be put on the first declaration as it's the first declaration that is being used to determine the type for the same named variable. This works for Mypy, Pyright and Pyre.

```python
        try:
            level: int | str = logger.level(record.levelname).name
        except ValueError:
            level = record.levelno
```

Reply:
@quantumpacket Is it considered best practice? I felt it might be confusing for the reader, so I decided to explicitly declare the `level` types before the assignment.

Reply:
Not sure what would consider it best practice or not, it's allowed by all type checkers listed. I guess it's a matter of preference at this point. :man_shrugging:

Reply:
Ok, I'll leave it as is then, but thanks for the heads-up.